spec/index.html

```diff
@@ -212,6 +212,7 @@ <h1>Structured Headers</h1>
             <li><b>for:</b> The type of value to which a clause of type "concrete method" or "internal method" applies.</li>
             <li><b>redefinition:</b> If "true", the name of the operation will not automatically link (i.e., it will not automatically be given an aoid).</li>
             <li><b>skip global checks:</b> If "true", disables consistency checks for this AO which require knowing every callsite.</li>
+            <li><b>skip return checks:</b> If "true", disables checking that the returned values from this AO correspond to its declared return type.</li>
```

Comment:
Instead of skipping return checks, shouldn't we be asserting that the return check fails? That way if it is later changed in such a way that it passes, we know we can remove the annotation.

Reply:
That would probably be better but I don't want to do the work to implement it. PRs welcome.

Reply:
OK actually that was trivial, done.